content/en/partners/cloud_cost_management/aws.md

```diff
@@ -0,0 +1,193 @@
+---
+title: AWS Cloud Cost for MSP Partners
+further_reading:
+  - link: "/cloud_cost_management/aws/"
+    tag: "Documentation"
+    text: "Cloud Cost Management for AWS"
+  - link: "/monitors/types/cloud_cost/"
+    tag: "Documentation"
+    text: "Create a Cloud Cost monitor"
+  - link: "/cloud_cost_management/tags/"
+    tag: "Documentation"
+    text: "Learn about Tags in Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/control-your-cloud-spend-with-datadog-cloud-cost-management/"
+    tag: "Blog"
+    text: "Gain visibility and control of your cloud spend with Datadog Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/cloud-cost-management-container-support/"
+    tag: "Blog"
+    text: "Understand your Kubernetes and ECS spend with Datadog Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/google-cloud-cost-management/"
+    tag: "Blog"
+    text: "Empower engineers to take ownership of Google Cloud costs with Datadog"
+  - link: "https://www.datadoghq.com/blog/total-cost-of-service-ownership-ccm/"
+    tag: "Blog"
+    text: "Quickly and comprehensively analyze the cloud and SaaS costs behind your services"
+    text: "Learn about Tags in Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/cloud-costs-study-learnings/"
+    tag: "Blog"
+    text: "Key learnings from the State of Cloud Costs study"
+  - link: "https://www.datadoghq.com/blog/unit-economics-ccm/"
+    tag: "Blog"
+    text: "Monitor unit economics with Datadog Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/finops-at-datadog/"
+    tag: "Blog"
+    text: "How we've created a successful FinOps practice at Datadog"
+  - link: "https://www.datadoghq.com/blog/cloud-cost-management-saved-millions/"
+    tag: "Blog"
+    text: "How we saved $1.5 million per year with Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/cloud-cost-management-oci/"
+    tag: "Blog"
+    text: "Manage and optimize your OCI costs with Datadog Cloud Cost Management"
+cascade:
+    algolia:
+      subcategory: 'Cloud Cost Management'
+      tags: ['cloud cost', 'cloud integrations', 'cloud cost management', 'cloud cost aws']
+
+---
+## Overview
+
+This guide explains how AWS Managed Service Providers (MSPs) configure AWS Billing Conductor and generate Pro Forma Cost and Usage Reports (CURs) so that a customer’s Member Account can be connected to Datadog CCM without exposing the MSP’s Management Account.
```

Comment:
```suggestion
This guide explains how AWS Managed Service Providers (MSPs) configure AWS Billing Conductor and generate Pro Forma Cost and Usage Reports (CURs) so that a customer's Member Account can be connected to Datadog CCM without exposing the MSP's Management Account.
```